frontend/survey-project/src/app/components/pages/survey/pages/answer-survey-page/answer-survey-page-routing.module.ts

```diff
@@ -0,0 +1,40 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { AnswerSurveyPageComponent } from './answer-survey-page.component';
+import { RouteNames } from '../../../../../shared/consts/routes';
+import { AnswerSurveyEmailComponent } from './ui/answer-survey-email/answer-survey-email.component';
+import { AnswerSurveyWebsiteComponent } from './ui/answer-survey-website/answer-survey-website.component';
+import { authGuard } from '../../../../../shared/guards/auth.guard';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: AnswerSurveyPageComponent,
+    children: [
+      {
+        path: RouteNames.AnswerSurveyEmailRoute,
+        component: AnswerSurveyEmailComponent
+      },
+      {
+        path: RouteNames.AnswerSurveyWebsiteRoute,
+        component: AnswerSurveyWebsiteComponent,
+        canActivate: [authGuard]
```

Comment:
Extension to the previous comment, I see that authentication checks are here, so maybe remove totally even the comment if the authentication policy is not connected to all of the SurveyModuleRoutes children.